t2.dart

```diff
@@ -0,0 +1,13 @@
+void switchInt() {
+  const int A = 1;
+  int v = 5;
+
+  switch (v) {
+    case 1:
+    case 2:
+    case A:
```

Comment:
The value of the case clause ('A') is equal to the value of an earlier case clause ('1').

```suggestion
```